download-dataset.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# Copyright 2019 David Bishai.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+N=499
+NUM_PROC=6
+
+download_and_compare() {
```

Comment:
"download_check_and_extract" would be a better name I guess

Reply:
agreed

Reply:
agreed